packages/skype.rb

```diff
@@ -14,24 +14,18 @@ class Skype < Package
     depends_on 'sommelier'
   end
 
-  binary_url ({
-    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/skype-8.56.0.103-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    x86_64: '1670a0c74ac08b224034aefbd27f3d03b89529e7168dc84ac095944a1aa93736',
-  })
-
   def self.build
-    system 'wget https://go.skype.com/skypeforlinux-64.deb'
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('skypeforlinux-64.deb') ) == '20b1b1b85c76cac9d224dccf5748cf21f954fe1a36c98fd5a7aa13d8448a1e07'
-    system 'alien -tc skypeforlinux-64.deb'
-    system "tar xvf skypeforlinux-#{version}.tgz"
+    system 'curl https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_8.65.0.78_amd64.deb -#o skype.deb'
+    puts "Archive downloaded".lightgreen
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('skype.deb') ) == '9026f043697d7d0783e7232701a868229d9a91e91dc0ae86509732c1c7bf0c13'
+    system 'alien -tc skype.deb'
+    system "tar xvf skypeforlinux-#{version}.tgz > /dev/null"
```

Comment:
I still don't like the `> /dev/null`.  On big archives, this will hang without any progress indication.

Reply:
Done